osidb/models.py

```diff
@@ -725,6 +725,24 @@ def _validate_major_incident_article(self):
                 "A flaw marked as Major Incident does not have an article."
             )
 
+    def _validate_major_incident_mitigation(self):
+        """
+        Tests that a Flaw that is Major Incident has a mitigation.
+        """
+        if self.is_major_incident and self.mitigation == "":
```

Comment:
I think that these alerts (both) should be rather non-blocking. I see that it is not clearly specified in the task and it has no SFM2 counterpart to compare. There is a similar alert checking that a major incident has summary (doc text) which is also non-blocking so I would consider this on the same level of restriction.

Reply:
I updated all article, mitigation and statement validations to use alerts.

Currently, `validate_major_incident_summary` uses exceptions instead of alerts. I will update this last one once the MI model is reworked (doing it now would not make sense).